BEIMA.Backend/MongoService/MongoConnector.cs

```diff
@@ -104,15 +104,26 @@ private void CheckIsConnected()
         /// <returns>BsonDocument that was requested</returns>
         private BsonDocument Get(ObjectId objectId, string dbName, string collectionName)
         {
-            CheckIsConnected();
-
             var filter = Builders<BsonDocument>.Filter.Eq("_id", objectId);
+            return GetFiltered(filter, dbName, collectionName)?.FirstOrDefault();
+        }
+
+        /// <summary>
+        /// Gets a single BsonDocument from the given database/collection, given an ObjectId.
```

Comment:
I think we should probably indicate that this can return any number of documents.

Reply:
fixed